packages/graphql-yoga/src/index.ts

```diff
@@ -4,14 +4,15 @@ export { type Plugin } from './plugins/types.js';
 export { type GraphiQLOptions } from './plugins/use-graphiql.js';
 export { renderGraphiQL, shouldRenderGraphiQL } from './plugins/use-graphiql.js';
 export { useReadinessCheck } from './plugins/use-readiness-check.js';
-export { useSchema } from './plugins/use-schema.js';
+export { type YogaSchemaDefinition, useSchema } from './plugins/use-schema.js';
 export * from './schema.js';
 export * from './server.js';
 export * from './subscription.js';
 export * from './types.js';
 export { maskError } from './utils/mask-error.js';
 export { type OnParamsEventPayload } from './plugins/types.js';
 export { createLRUCache } from './utils/create-lru-cache.js';
+export { mergeSchemas } from '@graphql-tools/schema';
```

Comment:
Why do you want to export this instead of importing it from `@graphql-tools/shcema` where you need it ?

Reply:
Yep, let's keep the export. Could be useful for other devs, and is one less dependancy in this plugin.